internal/pkg/manifest/svc.go

```diff
@@ -140,3 +164,19 @@ func intpcopy(v *int) *int {
 	}
 	return aws.Int(*v)
 }
+
+// Valid sidecar portMapping example: 2000/udp, or 2000 (default to be tcp).
+func parsePortMapping(s *string) (port *string, protocol *string, err error) {
```

Comment:
I really like the use of named return values when you have more than one return value with the same type! :clap: